src/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -177,4 +178,21 @@ class CSVUtils {
         }
         return Constants.DEFAULT_COLUMN_SEPARATOR;
     }
+
+    static String prependBOMToCSVString(String csvString) {
+        return '\uFEFF' + csvString
+    }
+
+    static String stripBOMIfPresent(String csvString) {
+        return csvString.replace("\uFEFF", "")
+    }
+
+
+    static String detectCSVCharset(File file) {
+        def detector = new UniversalDetector(null)
```

Comment:
Default constructor should be ok here, no?
https://github.com/albfernandez/juniversalchardet/blob/main/src/main/java/org/mozilla/universalchardet/UniversalDetector.java#L105